### DISCUSSION TITLE
Openshift container

### DISCUSSION THREAD

**Discussion by DanielMGoldberg** [gssapi/gss-ntlmssp#105] [Q&A] [answered]
Hi,
First of all, thank you for your hard work and contribution!
I would like to know if this package is compatible with my OpenShift Linux container. Currently, I'm encountering an "unsupported NTLM" message.

**Comment by simo5** [comment, marked as answer]
Sorry but I have no idea, in principle GSS-NTLMSSP is compatible with anything that uses a modern MIT's krb5 implementation and is properly installed (see install notes).

On servers though you will also need configuration to provide it with either hashes or a connection to a domain server (via winbindd).

In general NTLM authentication is something that should not be used anymore as it is insecure and broken. It is provided for legacy purposes only.
